jupyter_server/extension/application.py

```diff
@@ -174,7 +174,7 @@ def _default_open_browser(self):
 
     @classmethod
     def get_extension_package(cls):
-        return cls.__module__.split('.')[0]
+        return __import__(cls.__module__, fromlist=cls.__module__).__package__
```

Comment:
We can ask the module for its package name, in the case of a namespace package this includes the first dot.